js-intro.js

```diff
@@ -20,14 +32,28 @@ var temp3 = 212
 var myNumbers1 = [3, 7, 0, 6, -9]
 
 
+mult5 = (array) => {
+    let newArr = []
+    for (let i = 0; i<array.length; i++) {
+        newArr.push(array[i]*5)
+    }
+    return newArr
+}
 
+////Test:
+//console.log(mult5(myNumbers1));
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
-
+mult5map = (array) => {
+  let newArr = array.map(value => value*5)
```

Comment:
If you create a new variable that isn't being used anywhere else in your code, and you're only creating it to be returned later, you can also just say `return newArr = array.map(value => value*5)`. Your solution is not wrong, but it'll save you a line of code ;)